queries.sql

```diff
@@ -119,3 +119,27 @@ SELECT species, AVG(escape_attempts) AS avg_escapes
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
```

Comment:
- [ ] It would be better if we write the following queries as seen in the project requirements screenshot below
![Screenshot from 2023-05-03 20-32-37](https://user-images.githubusercontent.com/46388113/236041830-350110ff-cf60-4764-b34a-62f9335f180c.png)

Reply:
thank you for the reviews @addod19